test/gtest/ucs/test_datatype.cc

```diff
@@ -486,6 +487,17 @@ UCS_TEST_SKIP_COND_F(test_datatype, ptr_array_perf,
         ASSERT_TRUE(present);
     }
 
+    ucs_time_t foreach_start_time = ucs_get_time();
+    unsigned index;
+    void *element;
+    n_elements = 0;
+    ucs_ptr_array_for_each(element, index, &pa) {
+        void *ptr GTEST_ATTRIBUTE_UNUSED_;
+        int present = ucs_ptr_array_lookup(&pa, index, ptr);
+        ASSERT_TRUE(present);
+        n_elements++;
```

Comment:
n_elements doesn't seem to be doing anything useful.  `EXPECT_EQ(n_elements, count)` perhaps?

Reply:
Fixed.
Removed n_elements completely from this code...